src/SDKs/_metadata/network_resource-manager.txt

```diff
@@ -1,17 +0,0 @@
-﻿Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=D:\Projects\azure-sdk-for-net\tools\..\src\SDKs\
-2018-05-31 00:48:46 UTC
-1) azure-rest-api-specs repository information
-GitHub fork: Azure
-Branch:      master
-Commit:      a5af2bc46e91b4b6f6aa25408c1814532fcfc041
-
-2) AutoRest information
```

Comment:
Don't delete, check this out from the remote branch.

Reply:
Good catch, thanks. Done.